docs/plugin-transform-modules-umd.md

```diff
@@ -4,6 +4,10 @@ title: @babel/plugin-transform-modules-umd
 sidebar_label: transform-modules-umd
 ---
 
+This plugin transforms ES2015 modules to [UMD](https://github.com/umdjs/umd). Note that only the _syntax_ of import/export statements (`import "./mod.js"`) is transformed, as Babel is unaware of different resolution algorithms between implementations of ES2015 modules and UMD.
+
+:warning This plugin does not support dynamic import (`import('./lazy.js')`).
```

Comment:
Nit: the warning sign isn't properly displayed, I think that you should directly paste the emoji.